packages/manager/apps/hub/src/dashboard/dashboard.service.js

```diff
@@ -0,0 +1,10 @@
+export default class DashboardService {
+  /* @ngInject */
+  constructor($http) {
+    this.$http = $http;
+  }
+
+  ipFailover() {
```

Comment:
a good practice is to use a verb for function name, "ipFailover" doesn't mean much but "listIpFailover" or "getIpsFailover" provides more infos on what the function does

Reply:
Its's applied.